metagraph/src/graph/alignment/aligner_methods.hpp

```diff
@@ -124,7 +124,9 @@ class SuffixSeeder : public BaseSeeder {
     template <typename... Args>
     SuffixSeeder(Args&&... args)
```

Comment:
The implementation is in aligner_seeder_methods.cpp.
So, move these declarations to aligner_seeder_methods.hpp?